app/models/concerns/taggable.rb

```diff
@@ -1,63 +1,23 @@
 module Taggable
-
```

Comment:
Do we ever want to use Taggable without Filterable? Worth a comment here in that case.

I like keeping them as 2 separate modules even if Taggable is never used alone though.

Reply:
Bookmarks have tags without filters, so in theory that would be a class that's Taggable but not Filterable. But bookmarks don't divide up their tags by type, which means that pretty much all of the code in `taggable.rb` is useless for bookmarks. So the Bookmark class doesn't `include Taggable`.